src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details.html

```diff
@@ -293,12 +293,20 @@ <h5>MATCHES</h5>
                 </div>
                 <div id="match-accordion-item-{{match.control_number}}" class="panel-collapse collapse" ng-class="{ 'in': match.control_number == vm.selectedBestMatch }">
                   <div class="panel-body">
-                    <p ng-if="match.authors" class="authors">
-                      <span ng-repeat="author in match.authors">{{author.full_name}}
-                        <span ng-if="!$last"> ; </span>
-                      </span>
-                    </p>
+                    <div ng-if="match.authors" class="authors">
+                      <span ng-repeat="author in match.authors">{{author.full_name}} <span ng-if="!$last"> ; </span> </span>
+                    </div>
+                    <ul class="list-inline">
```

Comment:
this `ul` will show up no matter, how does it look when both `arxiv_eprint` and `number_of_pages` are missing?